lib/wibox/widget/slider.lua

```diff
@@ -271,6 +271,7 @@ local properties = {
     handle_width         = false,
     handle_border_width  = 0,
     handle_border_color  = false,
+    handle_widget       = false,
```

Comment:
Please add a `@property` section for this. You can copy/paste one and modify it. Also make sure to copy paste the corresponding file in `tests/examples/wibox/wiget/progressbar` and modify it to demonstrate how this property works. You need `ldoc` installed for this to work. You can open the doc in your local browser to see how it renders. No server required, they are static HTML files.

Reply:
I assume you meant `tests/examples/wibox/widget/slider`. I was looking through those while I was trying to get the documentation working. I couldn't get the properties to show up in the `Object Properties` header, and I assume it has something to do with this. What file exactly should I edit?